jobs/service-fabrik-broker/spec

```diff
@@ -181,6 +189,15 @@ properties:
   internal.domain_socket.path:
     description: "domain socket path used to communicate with broker monitoring agent"
     default: /tmp/sfevents
+  internal.floating_ip:
```

Comment:
Same here, declared here but used in keepalived. Why not declare there only, if it is not needed here?

Reply:
+1. Please move all props of keepalived in its job and if in case broker needs them then import them